### PR TITLE
[ROCm] add softmax, topk, layernorm to microbench

### PR DESCRIPTION
**Pull request by PeixuanZuo**
### Description

Add softmax, layernorm, topk benchmark to microbench.

### CODE REVIEW

File: onnxruntime/python/tools/microbench/softmax.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+
+class BenchmarkSoftmax(BenchmarkOp):
+    def __init__(self, args):
+        BenchmarkOp.__init__(self, args)
+
+    @classmethod
+    def create_inputs_outputs(cls, op_param):
+        np.random.seed(0)
+        a = np.random.rand(op_param.dim1, op_param.dim2, op_param.dim3).astype(op_param.data_type)
```
Comment (mindest) [inline onnxruntime/python/tools/microbench/softmax.py:29]: suggest using meaningful names instead of a, b, c, d. Likewise in `topk.py`.

**Review by mindest — COMMENTED** [review]
A few comments. Otherwise it looks good to me.

File: onnxruntime/python/tools/microbench/softmax.py (hunk 1 above)
Comment (PeixuanZuo) [reply onnxruntime/python/tools/microbench/softmax.py]: Done.

File: onnxruntime/python/tools/microbench/topk.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+class BenchmarkTopK(BenchmarkOp):
+    def __init__(self, args):
+        BenchmarkOp.__init__(self, args)
+
+    @classmethod
+    def create_inputs_outputs(cls, op_param):
+        np.random.seed(0)
+        a = np.random.rand(op_param.dim1, op_param.dim2, op_param.dim3).astype(op_param.data_type)
+        b = np.array([op_param.k]).astype(np.int64)
```
[the remark this answers is not in the document]
Comment (PeixuanZuo) [reply onnxruntime/python/tools/microbench/topk.py]: Done.

**Pull request merged**